crates/viewer/re_viewer/src/app_state.rs

```diff
@@ -214,6 +215,10 @@ impl AppState {
             )),
         );
 
+        // The root container cannot be dragged.
+        let undraggable_items =
+            re_viewer_context::Item::Container(viewport_ui.blueprint.root_container).into();
+
```

Comment:
This `undraggable_items` business feels very hacky, but I'm unsure how to factor that better. Feedback welcome.

Reply:
If we add the viewport blueprint to the the `ViewerContext`, then `handle_select_hover_drag_interactions` can check for it directly

Reply:
`re_viewport_blueprint` depends on `re_viewer_context`, so that cannot be done. And I would like to avoid merging these crates.